src/components/field-region/field-region.repository.ts

```diff
@@ -77,33 +83,40 @@ export class FieldRegionRepository extends DtoRepository(FieldRegion) {
     return await query.first();
   }
 
-  async readOne(id: ID, session: Session) {
+  async readOne(id: ID, session: Session): Promise<UnsecuredDto<FieldRegion>> {
```

Comment:
If the method's return type wasn't previously declared let's leave it as inferred. It usually doesn't add much since we are essentially declaring the DB return type in the method (which could not reflect reality). So it just adds more typing and another place to maintain.